sklearn/datasets/_base.py

```diff
@@ -578,6 +578,9 @@ def load_iris(*, return_X_y=False, as_frame=False):
             .. versionadded:: 0.20
 
     (data, target) : tuple if ``return_X_y`` is True
+        Returns a tuple of two ndarray of shape (n_samples, n_features)
+        A 2D array with each row representing one sample and each column
+        representing the features and/or target of a given sample.
```

Comment:
`y`  has shape `(n_samples,)`.

I think it's better to split the second sentence into 2 sentences, one for data, one for target and give the shape of each array explicitly and remove the shape info from the first sentence.

Reply:
Thanks! Done!